### PR TITLE
Adds carcinisoprojection jelly, a limb regrowth chem that is made from slime mutation toxin & blood

### PR DESCRIPTION
**Pull request by BlueWildrose**
## About The Pull Request

This adds a new reagent that regrows limbs, much like how slimepeople can regrow their limbs. It's made from the xenobiology exclusive slime mutation toxin, and blood. Essentially, if you're at that phase in xenobiology where you can turn people into slimepeople, you can make this chemical. Also much like the slimeperson limb regrowth ability, it will use blood to regrow the limbs upon consumption. It doesn't check for a minimum blood amount like the ability however, so any abusers should beware.

Overdosing gives a new special, nasty disease called "crab cancer", which must be cured by mutadone if it occurs. It deals nasty amounts of brute damage under a probability of 5 per tick, while causing flesh (read: meat and crab meat) to fall off of the victim of the disease. The more-human meat, much like how the gibber works, will have the victim's name on it. Could be used as a means to harvest meat from monkeys - there's always silver extracts if the xenobiologist doesn't care about meat otherwise.

## Why It's Good For The Game

Some people may want to regrow their limbs without having to change species.

This essentially is a chemical that removes the middleman with switching to a slime subspecies and back, and offers something that could be passed to the chemistry desk safely. It's balanced around what xenobiology currently is.

## Changelog
:cl:
add: Added carcinisoprojection jelly, a limb regrowth chemical made from xenobiology's slime mutation toxin & blood. Don't overdose!
/:cl:

### CODE REVIEW

**Comment by TheEaterOfBeans** [comment]
time to chug carcinisoprojection jelly and bicaridine so I can convert to Sarkicism

**Comment by bunny232** [comment]
I don't see the issue with this for it to be held for so long.

**Comment by BlueWildrose** [comment]
The maintainers have been a bit busy to review the more potentially controversial PRs from what I've understood, but they can take their time.

**Comment by BlueWildrose** [comment]
Looking over this though, maybe I put the overdose threshold a bit too high for giving crab cancer. Xenobiologists already have access to a sufficiently more powerful method of eliminating someone, and that's the advanced mutation toxin from the black slime extracts. Which, if you're already on the path of green slimes, that's only just one more evolution step for xenobiology to access before you have such a thing. You would only need a syringe gun with optional piercing rounds to inject it into others.

We'll see what the maintainers think when they come around for this. Eventually.

**Comment by TheEaterOfBeans** [comment]
Crab cancer should have a small chance to spawn actual crabs.

**Comment by Linzolle** [comment]
I don't know why this exactly needed further discussion or approval, but I personally am into it. Will probably merge this soon if nobody else says anything about it.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm
Comment (DeltaFire15) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Preserve parent proc check (it returning false should prevent all further actions) (don't commit this directly because tabs don't work well with this)
```suggestion
	. = ..()
	if(!.)
		return
	if(method == TOUCH) //as funny as it would be to have this hurled at someone, nah.
```

**Review by DeltaFire15 — CHANGES_REQUESTED** [review]
One requested change, plus probably add a check for whether the mob is dead and prevent it from working if true (as reaction_mob does not care about a mob's state)

**Comment by BlueWildrose** [comment]
I added a few extras like a mood debuff and a bounty inclusion so that should be everything.

File: code/datums/diseases/crabcancer.dm
Comment (DeltaFire15) [inline code/datums/diseases/crabcancer.dm]: No return needed here since it's already the end of the proc, should be good to merge other than that so I'll do that as soon as its gone.

**Review by DeltaFire15 — APPROVED** [review]

**Pull request merged**
